velox/docs/functions/presto/decimal.rst

```diff
@@ -130,23 +130,24 @@ difference in s1 and s.
 Like in Addition, the precision of the result may exceed 38. The choices are the
 same. Presto chooses to cap p at 38 and allow runtime errors.
 
-Let’s say `a` is of type decimal(p1, s1) with unscaled value `A` and `b` is of
-type decimal(p2, s2) with unscaled value B.
+Let’s say `a` is of type decimal(p1, s1) with unscaled value A, `b` is of
+type decimal(p2, s2) with unscaled value B, and the result `r` is of type
+decimal(p, s) with unscaled value R.
 
 ::
 
     a = A / 10^s1
     a = B / 10^s2
 
-The result type precision and scale are:
+For the division operation :code:`a / b`, the result type precision and scale
+are:
 
 ::
 
 	s = max(s1, s2)
 	p = p1 + s2 + max(0, s2 - s1)
 
-The result 'r' has 's' digits after the decimal point and unscaled value R. We
-derive the value of R as follows:
+We derive the value of R as follows:
```

Comment:
Why did you remove "The result 'r' has 's' digits after the decimal point and unscaled value R" ?